src/transformers/models/swinv2/modeling_swinv2.py

```diff
@@ -1099,8 +1099,8 @@ def forward(
     """,
     SWINV2_START_DOCSTRING,
 )
-# Copied from transformers.models.swin.modeling_swin.SwinForMaskedImageModeling with SWIN->SWINV2,Swin->Swinv2,swin->swinv2,224->256,window7->window8
```

Comment:
This means the forward is now decoupled from Swin, is this intended?

Reply:
Can't we just add the checkpoint name in the Copied from statement?

Reply:
Ok! It doesn't allow for a whole name to be replaced right? So I do it in pieces

Reply:
You ca put the whole name, the only thing it doesn't accept are spaces or commas.

Reply:
See an example [here](https://github.com/huggingface/transformers/blob/737bff6a36f05761a25444174eab986e028b2b94/src/transformers/models/longformer/tokenization_longformer.py#L115)